src/Autofac.Integration.Wcf/AutofacInstanceContext.cs

```diff
@@ -103,7 +105,17 @@ public AutofacInstanceContext(ILifetimeScope container)
             {
                 throw new ArgumentNullException("container");
             }
-            this.OperationLifetime = container.BeginLifetimeScope();
+            this.OperationLifetime = container.BeginLifetimeScope("WcfRequest", (builder) =>
```

Comment:
The name of the lifetime scope should be a constant rather than an inline magic string so folks can reuse it when registering things. The constants class name [in core Autofac is `MatchingScopeLifetimeTags`](https://github.com/autofac/Autofac/blob/ac840a25dd82b0db3429cd15403411e963bb2102/src/Autofac/Core/Lifetime/MatchingScopeLifetimeTags.cs).